sips/sip-025/sip-025-sbtc.md

```diff
@@ -0,0 +1,906 @@
+# Preamble
+
+SIP Number: 022
+
+Title: A Trustless 2-way Bitcoin Peg for Stacks
+
+Authors:
+* Aaron Blankstein <aaron@hiro.so>
+* Mike Cohen <mjoecohen@gmail.com>
+* Brice Dobry <brice@hiro.so>
+* Marvin Janssen
+* Friedger Müffke <mail@friedger.de>
+* Jesus Najera 
+* Jude Nelson <jude@stacks.org>
+* Don Park <donp@trustmachines.co>
+* Alie Slade <aslade@hiro.so>
+* Andre Serrano <andre@resident.stacks.org>
+* @Soju-Drinker 
+* Igor Sylvester <igor@trustmachines.co>
+* Joey Yandle <joey@trustmachines.co> 
+
+Consideration: Technical, Governance
+
+Type: Consensus
+
+Status: Draft
+
+Created: 28 September 2023
+
+License: CC0-1.0
+
+Sign-off:
+
+Layer: Consensus (hard fork)
+
+Discussions-To: https://github.com/stacksgov/sips
+
+# Abstract
+
+This SIP proposes the creation of a **trustless two-way peg** in which BTC (the
+tokens of the current Stacks burnchain) can be manifested on Stacks as a
+SIP-010-compliant fungible token, called sBTC, and later redeemed for the same
+number of BTC.  This proposal redefines the act of Stacking to be the act of
+maintaining a _peg wallet_, which is used to hold BTC and process sBTC
+redemptions.  Stackers are collectively responsible for ensuring that all
+liquid sBTC are backed by an equal number of BTC, and that redemptions are
+processed in a timely manner.  As a reward, they continue to receive PoX
+payouts.
+
+# Introduction
+
+The lack of a stateful smart contract system on Bitcoin necessitates the
+construction of systems in which a representation of locked Bitcoin is traded
+within a separate smart-contract-powered blockchain.  These systems aim to
+provide a "2-way peg" between the representation of the locked BTC ("wrapped
```

Comment:
[nit] consistently use two-way or 2-way, pick one. Also, why is "2-way peg" in quotes?

Reply:
It's a term of art being introduced in this SIP.  Will emphasize instead of quote.  Also, 2-way peg it is.